### PR TITLE
Feature: Add additional menu features to personal info screen (KIDS-1100)

### PR DESCRIPTION
**Pull request by TammiLion**
## Description

https://linear.app/givt/issue/KIDS-1100/add-additional-menu-features-to-personal-info-screen
EXCEPT BIOMETRICS
Adds dynamic theme switching.

## Summary by CodeRabbit

- **New Features**
  - Added dynamic theme switching within the app for a customizable user experience.
  - Introduced new routes and pages for enhanced navigation within the family app.
  - Added a new event for account termination success in analytics.

- **Bug Fixes**
  - Addressed minor UI inconsistencies by updating the app bar and various widgets.

- **Enhancements**
  - Improved logout functionality with additional logging and error handling.
  - Updated paths for various family app pages to streamline navigation.

- **Styling**
  - Introduced new button theme properties for a consistent look and feel across the app.

- **Chores**
  - Refined import statements for better modularization and maintainability of the codebase.

### CODE REVIEW

**Comment by TammiLion** [comment]
@coderabbitai can you review this PR?

**Comment by TammiLion** [comment]
@coderabbitai review

File: lib/shared/widgets/buttons/custom_green_elevated_button.dart [hunk 1]
```diff
@@ -8,6 +12,10 @@ class CustomGreenElevatedButton extends StatelessWidget {
   final VoidCallback? onPressed;
   @override
   Widget build(BuildContext context) {
+    if(kDebugMode) {
+      final isFamilyApp = AppThemeSwitcher.of(context).isFamilyApp;
+      log('CustomGreenElevatedButton isFamilyApp: $isFamilyApp');
+    }
     return Padding(
```
Comment (Daniela510) [inline lib/shared/widgets/buttons/custom_green_elevated_button.dart:19]: We already get logging when switching themes, so is this necessary? I guess no harm in it being there but just asking

File: lib/features/family/app/pages.dart [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
   parentAvatarSelection(
-      path: '/parent-avatar-selection', name: 'PARENT-AVATAR-SELECTION'),
-  wallet(path: '/wallet', name: 'WALLET'),
-  camera(path: '/camera', name: 'CAMERA'),
-  success(path: '/success', name: 'SUCCESS'),
+      path: 'parent-avatar-selection', name: 'PARENT-AVATAR-SELECTION'),
+  wallet(path: 'wallet', name: 'WALLET'),
+  camera(path: 'camera', name: 'CAMERA'),
+  success(path: 'success', name: 'SUCCESS'),
+  unregisterUS(path: 'unregister-us', name: 'UNREGISTER-US'),
```
Comment (Daniela510) [inline lib/features/family/app/pages.dart:10]: Im assuming removing the slashes because they are subroutes?

**Review by Daniela510 — COMMENTED** [review]
About givt page looks weird, i think its because of the new theme, idk if it should be fixed within scope of this?

**Review by Daniela510 — APPROVED** [review]
About givt page looks weird, i think its because of the new theme, idk if it should be fixed within scope of this?
Simulator Screenshot - eleven - 2024-06-24 at 16 45 33

File: lib/shared/widgets/buttons/custom_green_elevated_button.dart (hunk 1 above)
Comment (TammiLion) [reply lib/shared/widgets/buttons/custom_green_elevated_button.dart:19]: No not needed, will remove.

File: lib/features/family/app/pages.dart (hunk 2 above)
Comment (TammiLion) [reply lib/features/family/app/pages.dart:10]: Yes made them all subroutes.

File: lib/app/app.dart [hunk 3]
```diff
@@ -1,4 +1,7 @@
+import 'dart:developer';
```
[the remark this answers is not in the document]
Comment (TammiLion) [reply lib/app/app.dart:1]: Good bot.

File: lib/features/family/app/pages.dart [hunk 4]
```diff
@@ -1,14 +1,15 @@
 enum FamilyPages {
   profileSelection(path: '/profile-selection', name: 'PROFILE_SELECTION'),
   kidsAvatarSelection(
-      path: '/kids-avatar-selection', name: 'KIDS-AVATAR-SELECTION'),
+      path: 'kids-avatar-selection', name: 'KIDS-AVATAR-SELECTION'),
```
[the remark this answers is not in the document]
Comment (TammiLion) [reply lib/features/family/app/pages.dart:4]: Good bot.

**Pull request merged**